src/main/java/org/greekleanersinc/servicetemplate/client/ClientClient.java

```diff
@@ -0,0 +1,27 @@
+package org.greekleanersinc.servicetemplate.client;
```

Comment:
Major: WDYT about extracting client-related stuff to a separate module. This way it can be easily imported by other applications. More details in other comments

Reply:
It makes sense, divided the project into two modules (client and server)